src/components/dashboard/main-content/course-enrollments/data/hooks.js

```diff
@@ -191,14 +187,33 @@ export const useCourseUpgradeData = ({
       }
     };
 
+    // Don't do anything if the course is not part of the enterprise's catalog
+    if (!containsContentItems) {
+      setIsLoading(false);
+      return;
+    }
```

Comment:
[nit/suggestion] Nice, don't even need to call `fetchData` if we know the course run isn't in the catalog(s). I wonder if we should only _define_ `fetchData` after this check, too?